### PR TITLE
feat: optimized logits logic in nexa eval, 50 times faster when nexa …

### PR DESCRIPTION
**Pull request by zhycheng614**
…eval now

### CODE REVIEW

**Review by zhiyuan8 — APPROVED** [review]
@zhycheng614  please update this change in readme, emphasize we have supported 50X faster GGUF benchmark, and provide a sample code, versus original lm_eval
